source/api/data_api.rst

```diff
@@ -181,139 +181,138 @@ Search Imagery
-                                37.50141629,
-                                55.72561772
+                                69.3182,
+                                41.3647
                             ],
                             [
-                                37.50118307,
-                                55.48791481
+                                69.1906,
+                                41.3848
                             ],
                             [
-                                37.39506886,
-                                55.48708936
-                            ],
-                            [
-                                37.28895464,
-                                55.48626391
-                            ],
-                            [
-                                37.28866347,
-                                55.72563326
-                            ],
-                            [
-                                37.2883723,
-                                55.96500262
+                                69.2325,
+                                41.5352
                             ]
                         ]
                     ]
                 },
-                "pixelResolution": 0.31,
-                "acquisitionDate": "2021-07-07T08:42:03Z",
-                "productType": "Pan Sharpened Natural Color",
-                "sensor": "WV03_VNIR",
-                "colorBandOrder": "RGB",
+                "pixelResolution": 0.0,
+                "acquisitionDate": "2023-09-13T14:08:40Z",
+                "productType": "A",
+                "sensor": "JL1GF03D15",
+                "colorBandOrder": "unknown",
                 "cloudCover": 0.0,
-                "offNadirAngle": 6.471679
+                "offNadirAngle": -12.49,
+                "previewType": "png",
+                "previewUrl": "https://home.sat-imagery.com/geoserver/rs_data/wms?SERVICE=WMS&VERSION=1.1.0&REQUEST=GetMap&FORMAT=image%2Fpng&TRANSPARENT=true&LAYERS=rs_data%3AJL1GF03D15_PMS_20230913140838_200196307_108_0001_001_L1&STYLES=&serverType=geoserver&crossOrigin=anonymous&tiled=false&angle=0&WIDTH=188&HEIGHT=253&SRS=EPSG%3A3857&BBOX=7702262.3596810745%2C5066284.571753241%2C7721142.145319615%2C5091606.951481916",
+                "providerName": "sat_imagery"
             }
         ]
     }
 
-``aoi`` (required) - the geometry of the area (GeoJSON, Lat Lon coordinates) to search imagery for. Currently the only type ``Polygon`` is supported.
-
 .. important::
 
     The size of the area cannot exceed the size of processing AOI limit assigned to the specific user.
 
+
+    {aoi: geometry, - required, Geojson-like Polygon or Multipolygon acquisitionDateFrom: UTC time string acquisitionDateTo: UTC time stringminResolution: float, in metersmaxResolution: float, in metersmaxCloudCover: float, in percents minOffNadirAngle: float, in degrees maxOffNadirAngle: float, in degrees minAoiIntersectionPercent: float, in percents}
```

Comment:
Maybe it will be more readable in this form?

```suggestion
   {

   | ``aoi``: geometry, - required, Geojson-like Polygon or Multipolygon
   | ``acquisitionDateFrom``: UTC time string 
   | ``acquisitionDateTo``: UTC time string
   | ``minResolution``: float, in meters
   | ``maxResolution``: float, in meters
   | ``maxCloudCover``: float, in percents 
   | ``minOffNadirAngle``: float, in degrees
   | ``maxOffNadirAngle``: float, in degrees
   | ``minAoiIntersectionPercent``: float, in percents

   }
```

![image](https://github.com/Geoalert/mapflow-docs-en/assets/93107171/9144e949-f358-405d-9c64-f29b563170a3)

Reply:
> Maybe it will be more readable in this form?
> 
> ![image](https://user-images.githubusercontent.com/93107171/278834097-9144e949-f358-405d-9c64-f29b563170a3.png)

Thanks. I forgot to delete this. Fixed and looks better